src/configuration/connector.service.ts

```diff
@@ -0,0 +1,46 @@
+import { HttpService } from '@nestjs/axios';
+import { Injectable, Logger } from '@nestjs/common';
+import { AxiosResponse } from 'axios';
+import { VariableDefinitionsDto } from './dto/variable-definitions.dto';
+
+/**
+ * Service for connecting to individual service sidecars.
+ */
+@Injectable()
+export class ConnectorService {
+  private baseUrl:string
+  constructor(
+    private readonly logger: Logger,
+    private readonly httpService: HttpService,
+  ) {
+    this.baseUrl = 'http://localhost:3500/v1.0/invoke'
```

Comment:
why not directly at declaration?

Reply:
Changed in [2838e7949e949515df686773663b51b0a698984d](https://github.com/MiSArch/experiment-config-frontend/pull/1/commits/2838e7949e949515df686773663b51b0a698984d)